src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -250,6 +250,71 @@ def __init__(self, help_ctx, delimiters):
         super(CliHelpFile, self).__init__(help_ctx, delimiters)
         self.links = []
 
+        from knack.deprecation import resolve_deprecate_info, ImplicitDeprecated, Deprecated
+        from azure.cli.core.breaking_change import UpcomingBreakingChangeTag, MergedStatusTag
+        direct_deprecate_info = None
+        breaking_changes = []
+        deprecate_info = resolve_deprecate_info(help_ctx.cli_ctx, delimiters)
+        if isinstance(deprecate_info, Deprecated):
+            direct_deprecate_info = deprecate_info
+        elif isinstance(deprecate_info, UpcomingBreakingChangeTag):
+            breaking_changes.append(deprecate_info)
+        elif isinstance(deprecate_info, MergedStatusTag):
```

Comment:
Please add some comments to explain what's the ` MergedStatusTag`?

Reply:
Done